src/redux/actions/experiments/reorderSamples.js

```diff
@@ -37,9 +37,6 @@ const reorderSamples = (
       },
     );
 
-    const json = await response.json();
-    throwIfRequestFailed(response, json, endUserMessages.ERROR_SAVING);
```

Comment:
Another small fix: when the error handling was refactored this was missing so adapt it to the new error handling